rubygem-foreman_xen/rubygem-foreman_xen.spec

```diff
@@ -26,6 +26,7 @@ URL:        http://github.com/theforeman/foreman-xen
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires:   foreman-compute >= 1.5.0
+Requires:   foreman-compute < 1.8.0-0.1.RC1
```

Comment:
`< 1.8.0` will suffice

<pre>
$ rpmdev-vercmp 1.8.0 1.8.0-1
1.8.0 < 1.8.0-1
$ rpmdev-vercmp 1.8.0 1.8.0-0.1.RC1
1.8.0 < 1.8.0-0.1.RC1
</pre>

However this'll cause us problems with our repoclosure tests if I was to put it in 1.8 and nightly, since the dep won't resolve.  I could put it in 1.7 only, or leave this out for the time being until a new 1.8-compat release is available.  Preference?